ros_buildfarm/templates/snippet/archive_artifacts.xml.em

```diff
@@ -5,4 +5,5 @@
       <fingerprint>false</fingerprint>
       <defaultExcludes>true</defaultExcludes>
       <caseSensitive>true</caseSensitive>
+      <followSymlinks>true</followSymlinks>
```

Comment:
As far as I can tell this default setting did not previously require
a tag but now does. I think it's also worth considering _not_ following
symlinks for the archiver in this snippet as we only ever intend to
archive the generated archive from this build and no other contents
outside the workspace.

Reply:
Let's preserve current behavior for now. Follow up after we ship if you're still motivated.